src/actions/updateJobs.js

```diff
@@ -57,6 +58,51 @@ module.exports = ({ task, conn }) => async function findJobsBySponsor(params) {
     company: subscriber.companyName,
     subscriberId: subscriber._id
   })));
+  let messageObj = {channel: '#pro-notifications', blocks: []};
+  let company = '';
+  let jobTitle = '';
+  if(jobs.length > 1) {
```

Comment:
No need for this `if` statement, the only difference between the two `if` blocks is just "Job" vs "Jobs". Just leave the `jobs.length > 1` branch.